pkgs/applications/networking/browsers/webbrowser/default.nix

```diff
@@ -0,0 +1,108 @@
+{ stdenv, lib, fetchgit, makeDesktopItem, pkgconfig, makeWrapper
+# Build
+, python2, autoconf213, yasm, perl, ccache
+, unzip, gnome2, gnum4
+
+# Runtime
```

Comment:
```(suggestion)
```

Reply:
Hello, I don't understand why I should remove those comments ? They make things a bit more understandable

Reply:
It isn't necessary at all. 
By just reading the source code we can know what is needed at runtime and at build time. This isn't to be something mysterious.

This type of comment has the same effect as this:

```python
# adds one unit to variable X
X = X+1
```

Or
```C++
/*Calculates delta for a quadratic equation*/
delta = b*b-4*a*c
```

Reply:
I am unsure. I still want to organize the derivation needed like this instead of just putting them in any order. If I keep this organization but remove the comments, this would make less sense. I think those comments aren't harmful anyway, removing them don't give any special benefit.

Reply:
Comments are made to be informative, in order to convey some unusual insight about the code that can't be understood by just reading it.

And I don't think there is some metaphysical sense at all in the input list. 
It just a set of variables/attributes. There is nothing special in them, in order to be commented as "build" or "runtime".

I don't think it is a good programming style. But whatever. It looks like bikesheding.

Reply:
It is easier for me to follow dependencies in https://git.nuegia.net/webbrowser.git/tree/BUILD by doing this. And you said that you could figure out which derivations are needed at build time and at runtime by looking in buildInputs and nativeBuildInputs, but as you see this isn't the case. You may think those comments are "amateurish" but they personnally help me in organizing the derivation. So I won't remove them, since they really don't change anything.